lib/lists.nix

```
@@ -303,6 +303,18 @@ rec {
     else
       genList (n: first + n) (last - first + 1);
 
+  /* Return a list with `n` copies of an element.
+
+    Type: int -> a -> [a]
```

```suggestion
    Type: replicate :: int -> a -> [a]
```

This is the convention used for most functions